services/src/test/scala/cromwell/pipeline/service/GitLabProjectVersioningTest.scala

```diff
@@ -12,29 +14,42 @@ import org.mockito.Mockito.when
 import org.scalatest.concurrent.ScalaFutures
 import org.scalatest.{ AsyncWordSpec, Matchers }
 import org.scalatestplus.mockito.MockitoSugar
-import play.api.libs.json.Json
+import play.api.libs.json.OFormat
 
-import scala.concurrent.Future
+import scala.concurrent.{ ExecutionContext, Future }
 
 class GitLabProjectVersioningTest extends AsyncWordSpec with ScalaFutures with Matchers with MockitoSugar {
+  implicit val actorSystem: ActorSystem = ActorSystem()
```

Comment:
Creating your own actor system in test is a bad idea. Especially since you are not even closing it.

Reply:
What is the best practice in this case?